src/routes/sidebarmenu-component/sidebarmenu.jsx

```diff
@@ -0,0 +1,11 @@
+import SidebarMenu from '../../components/sidebarmenu/sidebarmenu'
+
+const SidebarMenuComponent = () => {
+  return (
+    <>
+      <SidebarMenu />
+    </>
+  )
+}
```

Comment:
Any specific reason for using empty fragments?

```suggestion
const SidebarMenuComponent = () => <SidebarMenu />
```

Reply:
I guess this is redundant. since I am returning a single component. I will remove this.